src/app/quiz/questions.js

```diff
@@ -0,0 +1,77 @@
+export const questions = [
+    {
+      question: 'What is JavaScript?',
+      options: [
+        'A programming language',
+        'A markup language',
+        'A database',
+        'An operating system'
+      ],
+      correct: 0
+    },
+    {
+      question: 'Which keyword is used to declare variables in JavaScript?',
+      options: ['var', 'let', 'const', 'All of the above'],
+      correct: 3
+    },
+    {
+      question: 'What is the result of 2 + "2" in JavaScript?',
+      options: ['4', '22', 'NaN', 'Error'],
+      correct: 1
+    },
+    {
+      question: 'What is the purpose of the "use strict" directive in JavaScript?',
+      options: [
+        'To enforce stricter parsing and error handling',
+        'To make the code run faster',
+        'To enable new JavaScript features',
+        'To prevent code execution'
+      ],
+      correct: 0
+    },
+    {
+      question: 'Which method removes the last element from an array?',
+      options: ['shift()', 'unshift()', 'pop()', 'push()'],
+      correct: 2
+    },
+    {
+      question: 'What is a closure in JavaScript?',
+      options: [
+        'A function that has access to variables in its outer scope',
```

Comment:
A function that creates a lexical scope to contain variables.

Reply:
thanks